CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and
 
 + Add - mkdocs documentation
 + Add - improved docstrings for mkdocs
++ Add - EXTRACT trigger and loader tools
```

Comment:
Please update the version here to `0.4.0`, update `version.py,` and add the following the end of this file:
```
[0.4.0]: https://github.com/datajoint/element-interface/releases/tag/0.4.0
```

Reply:
I'll keep it at 0.3.1 To be consistent with the previous unreleased changes at 0.3.1

Reply:
We should make a release with these updates.  And these changes warrant a bump to `0.4.0`.